content/common/language-guide-overview.md

```diff
@@ -0,0 +1,533 @@
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+{{ blockEnd }}
+
+{{ startTab "Go"}}
+
+**SDK: Outbound HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Key/Value**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound MySQL**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Postgres**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Custom (Cron)**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+{{ blockEnd }}
+
+{{ startTab "Javascript/Typescript"}}
+
+**SDK: Outbound HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Key/Value**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound MySQL**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Postgres**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Custom (Cron)**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+{{ blockEnd }}
+
+{{ startTab "Python"}}
+
+**SDK: Outbound HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Key/Value**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound MySQL**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Postgres**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**SDK: Outbound Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Custom (Cron)**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: HTTP**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Trigger: Redis**
+
+Local:
+
+Cloud:
+
+Documentation:
+
+Code Examples:
+
+Blog Article[s]:
+
+**Feedback**
+
+{{ blockEnd }}
+
+{{ startTab "Rust"}}
+
+**SDK: Outbound HTTP**
+
+Local: Yes
+
+Cloud: Yes
```

Comment:
Local vs cloud is not a language issue, it's a feature issue.  There's no circumstance in which a SDK would support a feature locally but not in cloud, or vice versa.  The SDK simply supports the feature or doesn't.